src/dcat_ap_no_validator_service/view/validator.py

```diff
@@ -57,8 +60,18 @@ async def post(self) -> web.Response:
             logging.error(traceback.format_exc())
             raise web.HTTPBadRequest(reason="Bad syntax in input graph.")
 
-        # Reply ok, all fields processed successfully
-        return web.Response(
-            body=results_graph.serialize(format="text/turtle"),
-            content_type="text/turtle",
-        )
+        # Try to content-negotiate:
+        format = "text/turtle"  # default
+        if accept_header == "*/*":

```

Comment:
Innføre støtte for kommaseparert liste?

Innføre http code 406 når accept header ikke er supportert?

Reply:
Linje 77 kaster 406, ref [web.HTTPNotAcceptable](https://docs.aiohttp.org/en/latest/web_exceptions.html). Eg kan godt slenge på ein kommentar som gjer det meir eksplisitt.

Reply:
Har pusha endring slik at liste der eitt av elementa er `*/*` er håndtert.